chartfx-math/src/test/java/de/gsi/math/spectra/ShortTimeFourierTransformTest.java

```diff
@@ -71,77 +70,67 @@ public void testRealSTFT() {
         fastFourierTrafo.realForward(vals1ref);
         final double[] vals1refMag = SpectrumTools.computeMagnitudeSpectrum(vals1ref, true);
         assertArrayEquals(vals1refMag, zValues[0]);
         // compare to transform t = 2 * step * dt
-        final double[] vals2ref = Arrays.copyOfRange(sine.getYValues(), 2 * step, 2 * step + nFft);
+        final double[] vals2ref = Arrays.copyOfRange(sine.getValues(DataSet.DIM_Y), 2 * step, 2 * step + nFft);
         Apodization.Hann.apodize(vals2ref);
         fastFourierTrafo.realForward(vals2ref);
         final double[] vals2refMag = SpectrumTools.computeMagnitudeSpectrum(vals2ref, true);
         assertArrayEquals(vals2refMag, zValues[2]);
         // compare to transform t = 5*step*dt
         final double[] vals3ref = new double[nFft];
-        System.arraycopy(sine.getYValues(), 5 * step, vals3ref, 0, nFft - 32 - step);
+        System.arraycopy(sine.getValues(DataSet.DIM_Y), 5 * step, vals3ref, 0, nFft - 32 - step);
         Apodization.Hann.apodize(vals3ref);
         fastFourierTrafo.realForward(vals3ref);
         final double[] vals3refMag = SpectrumTools.computeMagnitudeSpectrum(vals3ref, true);
         assertArrayEquals(vals3refMag, zValues[5]);
 
         // compare to transform t = 5*step*dt with ZOH
-        ShortTimeFourierTransform.real(sine, sineSpectrogram, nFft, step, Apodization.Hamming, Padding.ZOH, false,
-                true);
+        ShortTimeFourierTransform.real(sine, sineSpectrogram, nFft, step, Apodization.Hamming, Padding.ZOH, false, true);
         final double[][] zValuesZOH = getDataAsNestedArray(sineSpectrogram);
         final double[] vals3refZOH = new double[nFft];
-        System.arraycopy(sine.getYValues(), 5 * step, vals3refZOH, 0, nFft - 32 - step);
+        System.arraycopy(sine.getValues(DataSet.DIM_Y), 5 * step, vals3refZOH, 0, nFft - 32 - step);
         Arrays.fill(vals3refZOH, nFft - 32 - step, nFft, vals3refZOH[nFft - 32 - step - 1]);
         Apodization.Hamming.apodize(vals3refZOH);
         fastFourierTrafo.realForward(vals3refZOH);
         final double[] vals3refMagZOH = SpectrumTools.computeMagnitudeSpectrum(vals3refZOH, true);
         assertArrayEquals(zValuesZOH[5], vals3refMagZOH);
 
         // compare to transform t = 5*step*dt with ZOH
-        final DataSet sineSpectrogramMirror = ShortTimeFourierTransform.real(sine, null, nFft, step,
-                Apodization.Rectangular, Padding.MIRROR, true, false);
+        final GridDataSet sineSpectrogramMirror = ShortTimeFourierTransform.real(sine, null, nFft, step, Apodization.Rectangular, Padding.MIRROR, true, false);
         final double[][] zValuesMirror = getDataAsNestedArray(sineSpectrogramMirror);
         final double[] vals3refMirror = new double[nFft];
-        System.arraycopy(sine.getYValues(), 5 * step, vals3refMirror, 0, nFft - 32 - step);
+        System.arraycopy(sine.getValues(DataSet.DIM_Y), 5 * step, vals3refMirror, 0, nFft - 32 - step);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'DataSet.DIM_Y' due to existing static import 'de.gsi.dataset.DataSet.DIM_Y'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)